src/sbt-test/skip-sbt1.4/scalafixEnable/build.sbt

```diff
@@ -21,5 +21,5 @@ lazy val scala212 = project
 lazy val scala3 = project
   .in(file("scala3"))
   .settings(
-    scalaVersion := "3.0.0-RC3" // built-in support for semanticdb
+    scalaVersion := "3.3.1" // built-in support for semanticdb
```

Comment:
`sbt-1.5/*` was ignored on JDK21 until now. The fix was simply to get a compatible scala 3 version (per the [compatibility matrix](https://docs.scala-lang.org/overviews/jdk-compatibility/overview.html)).